src/pages/tabbed-dashboard/App.jsx

```diff
@@ -49,26 +49,51 @@ const App = () => {
   const chatWidgetRef = useRef();
   /** @type {React.MutableRefObject<Callbridge.Dashboard>} */
   const widgetRef = useRef();
+  /** @type {React.MutableRefObject<HTMLSelectElement>} */
+  const selectRef = useRef();
 
   const credentials = useSelector(state => state.credentials);
 
   const renderHideDashboardElementsButton = () => {
     const onHideDashboardElementsClick = () => {
       // Contact our support team on how to see the list of hidable elements
-      setHideDashboardElements([50,51,52,53])
+      setHideDashboardElements(hideDashboardElements ? undefined : [50, 51, 52, 53]);
     }
 
     return (
-      <button
-        type="button"
-        className={`${MenuButtonStyles.menuButton} ${MenuButtonStyles.right}`}
-        style={{ top: '130px' }} // why is this hardcoded?
-        onClick={onHideDashboardElementsClick}
-      >
-        Hide Dashboard Elements
-      </button>
+      <div>
+        <button
+          type="button"
+          className={`${MenuButtonStyles.menuButton} ${MenuButtonStyles.hideDashboardElementsButton} ${MenuButtonStyles.right}`}
+          onClick={onHideDashboardElementsClick}
+        >
+          {`${hideDashboardElements ? 'Show' : 'Hide'} Dashboard Elements`}
+        </button>
+      </div>
     )
   }
+  
+  const renderMultiSelect = () => {
+    const onMultiSelectChange = (event) => {
+      const options = Array.from(event.target?.options);
+      const selectedValues = options
+      .filter(option => option.selected)
+      .map(option => Number(option.value));
+  
+      setHideDashboardElements(selectedValues.length > 0 ? selectedValues : undefined);
+    }
+
+    return (
+      <div>
+        <select multiple onChange={onMultiSelectChange} ref={selectRef} className={`${MenuButtonStyles.menuButton} ${MenuButtonStyles.multiSelectHideDashboardElements} ${MenuButtonStyles.right}`}>
```

Comment:
It's currently synced in the below, but i'll move it to the `onMultiSelectChange` since it makes more sense.
```
useEffect(() => {
}, [hideDashboardElements]);
```

Reply:
Moved